### PR TITLE
Execution environment integration fixes.

### PR DESCRIPTION
**Pull request by MichaelMauderer**
### Pull Request Description

Follow up to  https://github.com/enso-org/enso/pull/6341#issuecomment-1522341055 . Contains some refactoring and solves some left over to-dos in the code. 

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [x] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

File: app/gui/view/graph-editor/src/shortcuts.rs [hunk 1]
```diff
@@ -62,7 +62,4 @@ pub const SHORTCUTS: &[(ensogl::application::shortcut::ActionType, &str, &str, &
     (Press, "debug_mode", "ctrl d", "debug_set_test_visualization_data_for_selected_node"),
     (Press, "debug_mode", "ctrl shift enter", "debug_push_breadcrumb"),
     (Press, "debug_mode", "ctrl shift up", "debug_pop_breadcrumb"),
-    (Press, "debug_mode", "ctrl n", "add_node_at_cursor"),
```
Comment (vitvakatu) [inline app/gui/view/graph-editor/src/shortcuts.rs:65]: Looks like accidental change.

File: app/gui/view/graph-editor/src/shortcuts.rs [hunk 2]
```diff
@@ -62,7 +62,4 @@ pub const SHORTCUTS: &[(ensogl::application::shortcut::ActionType, &str, &str, &
...
-    // === Execution Mode ===
-    (Press, "", "shift ctrl e", "toggle_execution_environment"),
```
Comment (vitvakatu) [inline app/gui/view/graph-editor/src/shortcuts.rs:67]: The sad thing that now merging this PR will make execution contexts impossible to use on Windows due to #6371 

However, I still like the simplification of the code.

Comment (vitvakatu) [reply app/gui/view/graph-editor/src/shortcuts.rs:67]: Perhaps we can add additional shortcuts to switch _to_ a specific environment? I imagine it should be a simple task. Like `ctrl+shift+l` for Live and `ctrl+shift+d` for design or whatever.

Comment (vitvakatu) [reply app/gui/view/graph-editor/src/shortcuts.rs:67]: Also, `shortcuts.md` needs to be updated

File: app/gui/view/graph-editor/src/shortcuts.rs (hunk 1 above)
Comment (MichaelMauderer) [reply app/gui/view/graph-editor/src/shortcuts.rs:65]: Fixed.

File: app/gui/view/graph-editor/src/shortcuts.rs (hunk 2 above)
Comment (MichaelMauderer) [reply app/gui/view/graph-editor/src/shortcuts.rs:67]: Added two shortcuts as discussed.

**Comment by vitvakatu** [comment]
## QA Report 🔴 

Shortcuts change the visual state of the dropdown but do not switch the execution environment. No messages to the engine and icons on nodes are not updated.

**Review by wdanilo — APPROVED** [review]

**Comment by MichaelMauderer** [comment]
> ## QA Report red_circle
> 
> Shortcuts change the visual state of the dropdown but do not switch the execution environment. No messages to the engine and icons on nodes are not updated.

This was a bug ion the dropdown not updating its output when using the FRP API to override the selection. Urgh. Should be fixed now.

**Comment by vitvakatu** [comment]
QA Passed 🟢

**Comment by vitvakatu** [comment]
Fixed recently spotted an issue with the environment name in generated code and retested the whole thing.

**Pull request merged**